addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -56,6 +57,7 @@ options.registry.gallery = options.Class.extend({
      * @override
      */
     onBuilt: function () {
+        this.addImages(false);
```

Comment:
Add a comment next to it `// TODO should consider the async parts`, I'll handle that someday